gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/CompoundStatementUtil.java

```diff
@@ -72,6 +75,29 @@ public static List<?> extractResourcesFromCompound(
         return List.of();
     }
 
+    public static List<?> extractResourcesFromCompoundWithoutBloodPressures(
```

Comment:
I feel a bit uneasy about having Domain Logic "What is/isn't a blood pressure" within this ComountStatementUtil class.

Feels like previously the relationship was pointing the other direction, i.e. "Domain logic" pointing at this dumb util class.